core/CMakeLists.txt

```diff
@@ -21,5 +21,7 @@ vecmem_add_library( vecmem_core core SHARED
    "include/vecmem/memory/host_memory_manager.hpp"
    "src/memory/host_memory_manager.cpp"
    "include/vecmem/memory/resources/memory_resource.hpp"
+   "src/memory/resources/terminal/malloc_resource.cpp"
+   "include/vecmem/memory/resources/terminal/malloc_resource.hpp"
```

Comment:
I would rather call these `include/vecmem/memory/host_memory.hpp` and `src/memory/host_memory.cpp`...